roles/seqreports/tasks/2_install_config.yml

```diff
@@ -0,0 +1,51 @@
+---
+
+- name: ensure seqreports config dir exists
+  file:
+    state: directory
+    path: "{{ seqreports_config_root }}"
+
+- name: deploy seqreports app config
+  template:
+    src: "templates/app.config.j2"
+    dest: "{{ seqreports_config_root }}/app.config"
+
+- name: deploy seqreports logger config
+  template:
+    src: "templates/logger.config.j2"
+    dest: "{{ seqreports_config_root }}/logger.config"
+
+- name: ensure seqreports log dir exists
+  file:
+    path: "{{ seqreports_log_dir }}"
+    state: directory
```

Comment:
similar to `seqreports_db_dir` below, you cannot create this directory here

Reply:
Ok, I write ticket